helpdesk/management/commands/create_escalation_exclusions.py

```diff
@@ -80,79 +24,70 @@ def handle(self, *args, **options):
 }
 
 
-def create_exclusions(days, occurrences, verbose, queues):
-    days = days.split(',')
-    for day in days:
-        day_name = day
-        day = day_names[day]
-        workdate = date.today()
-        i = 0
-        while i < occurrences:
-            if day == workdate.weekday():
-                if EscalationExclusion.objects.filter(date=workdate).count() == 0:
-                    esc = EscalationExclusion(
-                        name='Auto Exclusion for %s' % day_name, date=workdate)
-                    esc.save()
-
-                    if verbose:
-                        print("Created exclusion for %s %s" %
-                              (day_name, workdate))
-
-                    for q in queues:
-                        esc.queues.add(q)
-                        if verbose:
-                            print("  - for queue %s" % q)
-
-                i += 1
-            workdate += timedelta(days=1)
-
-
-def usage():
-    print("Options:")
-    print(" --days, -d: Days of week (monday, tuesday, etc)")
-    print(" --occurrences, -o: Occurrences: How many weeks ahead to exclude this day")
-    print(" --queues, -q: Queues to include (default: all). Use queue slugs")
-    print(" --verbose, -v: Display a list of dates excluded")
-
+class Command(BaseCommand):
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '-d',
+            '--days',
+            nargs='*',
+            choices=list(day_names.keys()),
+            required=True,
+            help='Days of week (monday, tuesday, etc)'
```

Comment:
Might be worth being a little more explicit about the format... something like:
'Days of week (monday, tuesday, etc). Enter the days as a space separted list.'

Reply:
Yes, good point, I updated help texts.